<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>201</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QComboBox" name="comboBox1">
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Mongolian Baiti&quot;;
padding: 2px;
border-radius: 5px;border: 1px solid grey;

</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox2">
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Mongolian Baiti&quot;;
padding: 2px
;border-radius: 5px;border: 1px solid grey</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox3">
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Mongolian Baiti&quot;;
padding: 2px;border-radius: 5px;border: 1px solid grey</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox4">
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Mongolian Baiti&quot;;
padding: 2px;border-radius: 5px;border: 1px solid grey</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox5">
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Mongolian Baiti&quot;;
padding: 2px;border-radius: 5px;border: 1px solid grey</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox6">
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Mongolian Baiti&quot;;
padding: 2px;border-radius: 5px;border: 1px solid grey</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>50</y>
      <width>91</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QSpinBox" name="spinBox1">
       <property name="styleSheet">
        <string notr="true">padding: 2px;
font: 10pt &quot;Mongolian Baiti&quot;;border-radius: 5px;border: 1px solid grey</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox2">
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Mongolian Baiti&quot;;
padding: 2px;border-radius: 5px;border: 1px solid grey</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox3">
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Mongolian Baiti&quot;;
padding: 2px;border-radius: 5px;border: 1px solid grey</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox4">
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Mongolian Baiti&quot;;
padding: 2px;border-radius: 5px;border: 1px solid grey</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox5">
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Mongolian Baiti&quot;;
padding: 2px;border-radius: 5px;border: 1px solid grey</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox6">
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Mongolian Baiti&quot;;
padding: 2px;border-radius: 5px;border: 1px solid grey</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>50</y>
      <width>91</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLineEdit" name="lineEdit1">
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Mongolian Baiti&quot;;
padding: 2px;border-radius: 5px;border: 1px solid grey</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit2">
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Mongolian Baiti&quot;;
padding: 2px;border-radius: 5px;border: 1px solid grey</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit3">
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Mongolian Baiti&quot;;
padding: 2px;border-radius: 5px;border: 1px solid grey</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit4">
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Mongolian Baiti&quot;;
padding: 2px;border-radius: 5px;border: 1px solid grey</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit5">
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Mongolian Baiti&quot;;
padding: 2px;border-radius: 5px;border: 1px solid grey</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit6">
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Mongolian Baiti&quot;;
padding: 2px;border-radius: 5px;border: 1px solid grey</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Mongolian Baiti&quot;;
background-color: rgba(0,0,0,0%);
color:black;
outline:2px black;
</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Exercises&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Mongolian Baiti&quot;;
background-color: rgba(0,0,0,0%);
color:black;
outline:2px black;
</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Reps&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1031</width>
      <height>581</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>1101</width>
      <height>771</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#widget{
background-color:qlineargradient(spread:pad, x1:0.108, y1:0.170455, x2:0.795, y2:0.835, stop:0 rgba(85, 255, 255, 255), stop:0.994318 rgba(255, 170, 255, 255));};</string>
    </property>
    <widget class="QCalendarWidget" name="calendarWidget">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>340</y>
       <width>411</width>
       <height>211</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Mongolian Baiti&quot;;
</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_add">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>290</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Mongolian Baiti&quot;;
border-style: outset;
border-width: 3px;
border-radius: 15px;
border: 1px solid grey;
padding: 6px;
background-color:qlineargradient(spread:pad, x1:0.108, y1:0.170455, x2:0.795, y2:0.835, stop:0 rgba(255, 170, 255, 255), stop:1 rgba(85, 255, 255, 255));</string>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_date">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>290</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Mongolian Baiti&quot;;
border-style: outset;
border-width: 3px;
border-radius: 15px;
border: 1px solid grey;
padding: 6px;
text-align: center;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_username">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>270</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Mongolian Baiti&quot;;
background-color: rgba(0,0,0,0%);
color:black;
outline:2px black;
</string>
     </property>
     <property name="text">
      <string>Enter date</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_search">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>290</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Mongolian Baiti&quot;;
border-style: outset;
border-width: 3px;
border-radius: 15px;
border: 1px solid grey;
padding: 6px;
background-color:qlineargradient(spread:pad, x1:0.108, y1:0.170455, x2:0.795, y2:0.835, stop:0 rgba(255, 170, 255, 255), stop:1 rgba(85, 255, 255, 255));</string>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_clear">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>290</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Mongolian Baiti&quot;;
border-style: outset;
border-width: 3px;
border-radius: 15px;
border: 1px solid grey;
padding: 6px;
background-color:qlineargradient(spread:pad, x1:0.108, y1:0.170455, x2:0.795, y2:0.835, stop:0 rgba(255, 170, 255, 255), stop:1 rgba(85, 255, 255, 255));</string>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>60</y>
       <width>201</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">;font: 12pt &quot;Mongolian Baiti&quot;;
text-align: center;
border: 1px solid grey;
</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox7">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>200</y>
       <width>199</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Mongolian Baiti&quot;;
padding: 2px;
border-radius: 5px;border: 1px solid grey
</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_list">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>10</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;Mongolian Baiti&quot;;
background-color: rgba(0,0,0,0%);
color:black;
outline:2px black;
</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;You've trained on:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>10</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;Mongolian Baiti&quot;;
background-color: rgba(0,0,0,0%);
color:black;
outline:2px black;
</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Weight&lt;/span&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;(kg)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>60</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_list_2">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>10</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;Mongolian Baiti&quot;;
background-color: rgba(0,0,0,0%);
color:black;
outline:2px black;
</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Personal bests:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>60</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>60</y>
       <width>91</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLineEdit" name="lineEdit_pbSquat">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Mongolian Baiti&quot;;
padding: 2px;
border-radius: 5px;
border: 1px solid grey;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_pbBench">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Mongolian Baiti&quot;;
padding: 2px;
border-radius: 5px;
border: 1px solid grey;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_pbDeadlift">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Mongolian Baiti&quot;;
padding: 2px;
border-radius: 5px;
border: 1px solid grey;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_5">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>60</y>
       <width>91</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLabel" name="label_list_3">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Mongolian Baiti&quot;;
background-color: rgba(0,0,0,0%);
color:black;
outline:2px black;
</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Squat &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_list_4">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Mongolian Baiti&quot;;
background-color: rgba(0,0,0,0%);
color:black;
outline:2px black;
</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Bench press &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_list_5">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Mongolian Baiti&quot;;
background-color: rgba(0,0,0,0%);
color:black;
outline:2px black;
</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Deadlift &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="PlotWidget" name="graphWidget" native="true">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>270</y>
       <width>471</width>
       <height>281</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid grey;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_plot">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>200</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Mongolian Baiti&quot;;
background-color: rgba(0,0,0,0%);
color:black;
outline:2px black;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <zorder>widget</zorder>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>verticalLayoutWidget_2</zorder>
   <zorder>verticalLayoutWidget_3</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPower">
    <property name="title">
     <string>Power</string>
    </property>
    <addaction name="actionLog_out"/>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menuPower"/>
  </widget>
  <action name="actionLog_out">
   <property name="text">
    <string>Log out</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
